server/views/formPages/incident/relocationAndInjuries.html

```diff
@@ -52,6 +49,12 @@ <h1 class="govuk-heading-xl mainHeading">{{ pageTitle }}</h1>
         errorMessage: errors | findError('relocationType'),
         options: data.types.RelocationType | list | extractAttr('value') | toOptions('value', 'label')
       },
+      followUpQuestionForTypeOfRelocation: {
```

Comment:
Better to have clear boundaries between non generic and generic code.

On the surface this seems to be a generic macro `radiosWithNestedRadiosWithNestedOtherFieldTextbox`
but the implementation details/macro options are specific to one specific use case.

Reply:
There's no harm extracting out a very specific macro if it makes the implementation simpler. 

It made sense to make the other macros generic as we were planning to reuse them but in this case it might produce a more maintainable solution if we hardcoded values in the implementation. 

We could look at making the macro generic if it turned out we had more than one `radiosWithNestedRadiosWithNestedOtherFieldTextbox`?